pkg/api/v1/routes/handlers.go

```diff
@@ -241,6 +241,8 @@ func (r *Routes) serverFacilityCode(ctx context.Context, serverID uuid.UUID) (st
 func (r *Routes) exclusiveNonFinalConditionExists(ctx context.Context, serverID uuid.UUID) error {
 	otelCtx, span := otel.Tracer(pkgName).Start(ctx, "Routes.exclusiveNonFinalConditionExists")
 	defer span.End()
+	// XXX: Clean me up! We're looking for any active condition here, so don't list, do a store lookup
+	// here. Any active condition here is enough to block queuing a new one.
```

Comment:
Note: we'd only want active `Exclusive` conditions to block queueing a new one.

Reply:
All actions should be exclusive, especially once we start queuing a group of them at once.